webui/react/src/hooks/useModal/useModalExperimentCreate.tsx

```diff
@@ -159,7 +160,7 @@ const useModalExperimentCreate = (props?: Props): ModalHooks => {
           </Form.Item>
           {!isFork && (
             <Form.Item
-              label={`Max ${getMaxLengthType(config)}`}
+              label={`Max ${getMaxLengthType(config) ?? 'length'}`}
```

Comment:
**nit:** for this one `||` might be slightly "better" than `??`. `||` will trigger if the value is an empty string or other non-truthy values such as zero, null and undefined. Where as `??` will only trigger if the value is `null` or `undefined` only.

Reply:
I would argue that the only legal values of "getMaxLengthType" used to be `{"epochs", "records", "batches"}` but has been expanded to `{"epochs", "records", "batches", undefined}`, so both cases are equally correct for handling all valid outputs.  I can make that change anyway if you prefer it though.